packages/common/src/search/types/IFacet.ts

```diff
@@ -18,13 +18,20 @@ export interface IFacet {
   state?: "open" | "closed";
   // Individual options for this Facet
   options?: IFacetOption[];
-  // Operation when combining options. Only applies to `multi-select` and defaults to "OR"
-  operation?: ("OR" | "AND")[];
+  // Operation when combining the filter options.
+  // Implemetations should default to "OR" if not specified
+  operation?: "OR" | "AND";
+  /**
+   * This is still needed so we can direct the
+   * search to the correct api
+   * TODO: remove optional at breaking change
+   */
+  filterType?: "item" | "group" | "user" | "event";
   // Number of facet options to show by default. Only applies to `multi-select`, and defaults to all.
   optionLimit?: number;
 
   // DEPRECATE
```

Comment:
so these properties below will be removed?

Reply:
Yes. At the next breaking change